.gitattributes

```diff
@@ -0,0 +1,7 @@
+# Declare scripts  that will always have LF line endings on checkout.
+build text eol=lf
+cmd.sh text eol=lf
+lfs-image text eol=lf
+
+# Declare win dump file that will always have CRLF line endings on checkout.
+read.me text eol=crlf
```

Comment:
Hmm, why is that so i.e. why is that one different?

Reply:
I thought it would be printed on WinX mainly which likes crlf better than linux style lf. 
But wasn't  to sure about that.

Reply:
Good thinking, I never thought about this as I'm not on Windows. You obviously are?

> I thought it would be printed on WinX mainly

No idea if this assumption is correct or not but I understand your reasoning. I'm a proponent of "form follows function". So, the readme just has to be legible enough on all platforms so that users can easily extract the relevant information. This seems fulfilled here.